### DISCUSSION TITLE
Some issues regarding flow field inversion

### DISCUSSION THREAD

**Discussion by 15175829310** [mdolab/dafoam#665] [General]
May I ask if it is possible for the calculation domain of RANS to be larger than that of DNS data?
And does this FFD box have to wrap the entire RANS computational domain? Is there any way to solve this problem well.

**Comment by friedenhe** [comment]
1. Yes, the RANS domain can be larger than the DNS domain. You can select your reference points to be the DNS domain.
2. You do not need to have a FFD box for field inversion, check the scripts in tutorials/ramp/steady/train
